packages/protoc-gen-es/README.md

```diff
@@ -124,11 +124,26 @@ By default, we generate JavaScript and TypeScript declaration files, which
 produces the smallest code size. If you prefer to generate TypeScript, use
 `target=ts`.
 
+### `import_extension=.js`
+
+By default, [protoc-gen-es](https://www.npmjs.com/package/@bufbuild/protoc-gen-es)
+(and all other plugins based on [@bufbuild/protoplugin](https://www.npmjs.com/package/@bufbuild/protoplugin))
+uses a `.js` file extensions in import paths, even in TypeScript files.
+
+This is unintuitive, but necessary for [ECMAScript modules in Node.js](https://www.typescriptlang.org/docs/handbook/esm-node.html).
+Unfortunately, not all bundlers and tools have caught up yet, and Deno 
+requires `.ts`. With this plugin option, you can replace `.js` extensions 
+in import paths with the given value. For example, set
+
+- `import_extension=` to remove the `.js` extension
```

Comment:
@mfridman is suggesting a more explicit `import_extension=none`.

Reply:
Updated in https://github.com/bufbuild/protobuf-es/pull/317/commits/374cfbef890bdfc16f7830748d047499adbefb59